lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -15,14 +15,17 @@
 `@trace_charm(tracing_endpoint="my_tracing_endpoint")`
 
 2) add to your charm a "my_tracing_endpoint" (you can name this attribute whatever you like) **property**
-that returns an otlp grpc endpoint url. If you are using the `TracingEndpointProvider` as
+that returns an otlp http endpoint url. If you are using the `TracingEndpointProvider` as
```

Comment:
perhaps worth clarifying here that it can be a http or https url.

Reply:
True. Updated the comment here.

Reply:
is it me, or I don't see the update?

Reply:
Seems like this comment landed on a subset of commits: when you pick all commits, it's there: https://github.com/canonical/tempo-k8s-operator/pull/55/files#diff-7ad9005cca2d3157d4ec395255a4a9ec78900fb4a91288d532c24cd64fc74176R18